core/app.go

```diff
@@ -95,6 +96,8 @@ func NewApp(configFlag string) (*App, error) {
 	if err = cfg.InitLogging(); err != nil {
 		return nil, err
 	}
+	configJSON, _ := json.Marshal(*cfg)
```

Comment:
If you'd prefer I could make this something like this instead:

```go
if strings.ToLower(cfg.LogConfig.Level) == "debug" {
    configJSON, _ := json.Marshal(*cfg)
    log.Debugf("Loaded config: %v", string(configJSON))
}
```

So that we only marshal if we're in debug mode. I'm not sure how much you care about the extra overhead of this.